charts/internal/shoot-later-components/values.yaml

```diff
@@ -0,0 +1,7 @@
+rook-ceph:
+  enabled: true
+  image:
+    repository: rook/ceph
+    tag: v1.7.3
+
+namespace: rook-ceph # the namespace in which to install it, moral equivalent of "helm install --namespace"
```

Comment:
I was trying to get out of hard-coding it. I wanted it somewhere in the config or chart, so that it can be consumed from there. Hard-coding is mostly bad anyways.

Maybe I will put it in the provider config?